proto/stat_var.proto

```diff
@@ -247,25 +247,30 @@ message GetStatVarSummaryResponse {
 }
 
 message GetStatVarMatchRequest {
+  // Optional query from the user.
+  string query = 1;
```

Comment:
I don't see this field is used yet?

Reply:
Not yet but I was thinking of offering a generic interface later on. Shall I remove this for the time being?

Reply:
Adding when using it sounds good.

Reply:
Removed.